MANIFEST.in

```diff
@@ -18,3 +18,5 @@ include pytest.ini
 include tox.ini
 include setuptools/tests/config/setupcfg_examples.txt
 global-exclude *.py[cod] __pycache__
+prune dist
+prune build
```

Comment:
I'm surprised this behavior isn't the default. Is it the case that if a dist directory existed prior to this change, it would end up in the sdist? That seems like bad default behavior and would have come up in other projects. I'd like to avoid this concern if possible. It's another case where adopting setuptools_scm will also bypass the concern.

Reply:
We probably don't need this one.